src/pages/event.js

```diff
@@ -298,7 +298,7 @@ function TableContainer({tokens, ensNames, pageCount: pc, loading}) {
       <div className={`mobile-row open`}>
         <span className='id-title'>POAP ID</span><span className='id-content'>#{token.id}</span>
         <span className='address-title'>Address</span><span className='address-content ellipsis'>
-        <a href={`${POAP_APP_URL}/scan/${token.owner.id}`} target="_blank" rel="noopener noreferrer">{shrinkAddress(address, 15)}</a></span>
+        <a href={`${PRYSM_APP_URL}/profile/${token.owner.id}/achievements`} target="_blank" rel="noopener noreferrer">{shrinkAddress(address, 15)}</a></span>
```

Comment:
Move this URL to a function to reuse it on lines 349 and 372 as well

Reply:
done!